packages/larva-css/src/03-utilities/u-width.common.inline.scss

```diff
@@ -53,11 +53,30 @@ $widths_map: (
 	}
 }
 
+.lrv-u-width-100\@mobile-max {
```

Comment:
The breakpoints in utility files need to be organized in order of the breakpoints so that the desktop-xl will override desktop, for example. So, organize all mobile-max breakpoints at the top, tablet (if it exists), then desktop, followed by desktop-xl.

Reply:
@laras126 Fixed it.